packages/@okta/vuepress-site/docs/guides/oin-oidc-guide/protocol-level-requirements/index.md

```diff
@@ -2,52 +2,98 @@
 title: Protocol-level requirements
 ---
 
-### Code flow
+### Authorization Code flow
 
-For OIDC applications destined for the OIN, you are building a "web" application with a dedicated server-side back end that is capable of securely storing a Client Secret and exchanging information with an authorization server through trusted back-channel connections.
+For OIDC applications destined for the OIN, Okta recommends building a "web" application with a dedicated server-side back end capable of securely storing a Client Secret and exchanging information with an authorization server through trusted back-channel connections.
```

Comment:
Please leave "that is" in. It is helpful in translation to make **that** a dependent clause and it cues the translator that **capable** modifies **back end**

```suggestion
For OIDC applications destined for the OIN, Okta recommends building a web application with a dedicated server-side back end that is capable of securely storing a Client Secret and exchanging information with an authorization server through trusted back-channel connections.
```